recipes/tempo2/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "tempo2" %}
+{% set version = "2020.07.1" %}
+
+package:
+    name: "{{ name|lower }}"
+    version: "{{ version }}"
+
+source:
+    # eg, https://bitbucket.org/psrsoft/tempo2/downloads/tempo2-2019.01.1.tar.gz
+    #url: "https://bitbucket.org/psrsoft/{{ name }}/downloads/{{ name }}-{{ version }}.tar.gz"
+    ## This is v2020.07.1 but not tagged
+    url: "https://bitbucket.org/psrsoft/{{ name }}/get/b054862c0788.tar.gz"
```

Comment:
I believe that a quoted URL breaks the updating bot. You don't need the quotes there.

Reply:
Fixed now, thanks